spec/acceptance/init_spec.rb

```diff
@@ -1,15 +1,21 @@
 # run a test task
 require 'spec_helper_acceptance'
 
+username = if fact('operatingsystem') == 'windows'
+             'Administrator'
```

Comment:
May not be an issue but Administrator is not guaranteed to exist e.g. on German Windows it is ` Administratoren`

Reply:
https://tickets.puppetlabs.com/browse/PUP-6499

Reply:
Hey @glennsarti,

I am currently just putting in a fix for the following CI failure (FM-6924) and the specific username is only for test purposes. If you have any objections for merging please let me know.

PS - Thats really interesting, would `Administrator` differ for every language eg Japanese, Spanish, German etc?

Reply:
I don't have any objects as I _assume_ you'll be testing on US English for the time being.  Just note that we've been bitten before.

Reply:
Noted :) We'll keep an eye out.